data/features/mmex.json

```diff
@@ -59,13 +59,13 @@
         },
         {   "name": "crossplatform",
             "icon": "fas fa-desktop",
-            "title": "Cross Platform",
-            "text": "We provide builds for the most common operating systems: Windows, MacOS and Linux Ubuntu. It is possible to use it also on other OS building it directly from source code."
+            "title": "Cross-Platform",
+            "text": "We provide builds for the most common operating systems: Windows, macOS and Linux. It is possible to use it also on other operating systems building MMEX directly from source code."
         },
         {   "name": "opensource",
             "icon": "fab fa-github",
-            "title": "Free & Open Source",
-            "text": "OpenSource software, free download without ads and charges. It can be freely used, changed, and shared by anyone."
+            "title": "Free and Open-Source",
+            "text": "Open-source software, ad-free and gratis download. It can be freely used, changed, and shared by anyone."
```

Comment:
Requires a hyphen.
free as in freedom or free as in money? Both?

https://en.wikipedia.org/wiki/Open-source_software